### PR TITLE
[MKP] Remove service account credential from deployment page.

### PR DESCRIPTION
**Pull request by numerology**
Per offline discussion. The current way of generating and specifying SA credential brings additional security risk. We temporarily remove it from our MKP deploy page and users can inject their credential after deployment.

Will update the guidance and doc in another PR.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/2308)

### CODE REVIEW

File: manifests/gcp_marketplace/schema.yaml [hunk 1]
```diff
@@ -100,20 +100,23 @@ properties:
...
-      This deployment requires a service account to use for authentication when calling other GCP services. 
-      Specify the base64-encoded credentials for the service account you want to use. 
-      You can get these credentials by running the following command in a terminal window. 
-      \"$ gcloud iam service-accounts keys create application_default_credentials.json --iam-account [your-service-account] && cat application_default_credentials.json | base64\"
-      If you are using Linux, please use \"base64 -w 0\" to disable line wrapping.
-    type: string
-    default: ""
-    x-google-marketplace:
-      type: STRING
+# TODO: temporarily remove sa credential from deploy page in MKP. SA will be added by user manually
```
Comment (james-jwu) [inline manifests/gcp_marketplace/schema.yaml]: I think we can permanently remove SA credential box, right? The long term plan is to have our own configuration page. I don't see us bringing back SA credential box.

Comment (numerology) [reply manifests/gcp_marketplace/schema.yaml]: Got it. Done.

**Pull request merged**